dotnet/src/Connectors/Connectors.Memory.AzureCosmosDBNoSQL/AzureCosmosDBNoSQLVectorStore.cs

```diff
@@ -54,12 +54,16 @@ public IVectorStoreRecordCollection<TKey, TRecord> GetCollection<TKey, TRecord>(
                 vectorStoreRecordDefinition);
         }
 
-        var directlyCreatedStore = new AzureCosmosDBNoSQLVectorStoreRecordCollection<TRecord>(
```

Comment:
We should also allow AzureCosmosDBNoSQLCompositeKey above on line 44 where we check for the key type.
Sorry for putting the comment here, for some reason github is suddenly not allowing me to add comments to lines that haven't changed when viewing updates.

Reply:
Great catch, thank you! I've added additional check and added unit test to verify that composite key works correctly.